packages/amplify-graphql-default-value-transformer/src/graphql-default-value-transformer.ts

```diff
@@ -31,16 +31,24 @@ import { TypeValidators } from './validators';
 
 const nonStringTypes = ['Int', 'Float', 'Boolean', 'AWSTimestamp', 'AWSJSON'];
 
-const validateFieldType = (ctx: TransformerSchemaVisitStepContextProvider, type: TypeNode): void => {
+const validateFieldType = (ctx: TransformerSchemaVisitStepContextProvider, config: DefaultValueDirectiveConfiguration): void => {
+  const type = config.field.type;
+  const argc = config.directive.arguments!.length;
```

Comment:
Is `arguments` guaranteed to be non-null? If you can't guarantee it, it's safer to say 

```suggestion
  const argc = config.directive.arguments?.length ?? 0;
```

Reply:
Applied changes